forge-gui/res/cardsfolder/upcoming/resourceful_defense.txt

```diff
@@ -4,6 +4,5 @@ Types:Enchantment
 T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Permanent.YouCtrl+HasCounters | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever a permanent you control leaves the battlefield, if it had counters on it, put those counters on target permanent you control.
 SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Permanent.YouCtrl | TgtPrompt$ Select target permanent you control | CounterType$ EachFromSource | EachFromSource$ TriggeredCardLKICopy
 DeckHas:Ability$Counters 
-A:AB$ Pump | Cost$ 4 W | ValidTgts$ Permanent.YouCtrl | TgtPrompt$ Select target permanent you control | SubAbility$ DBMove | SorcerySpeed$ True | StackDescription$ SpellDescription | SpellDescription$ Move a counter from target permanent you control onto another target permanent. Activate only as a sorcery.
-SVar:DBMove:DB$ MoveCounter | Source$ ParentTarget | ValidTgts$ Permanent | TargetMin$ 2 | TargetMax$ 2 | TgtPrompt$ Select target permanent to move counter to | TargetUnique$ True | CounterType$ Any | CounterNum$ Any | StackDescription$ None
+A:AB$ MoveCounter  | Cost$ 4 W | Source$ ParentTarget | ValidTgts$ Permanent | TargetMin$ 2 | TargetMax$ 2 | TgtPrompt$ Select target permanent to move counter to | TargetUnique$ True | CounterType$ Any | CounterNum$ Any | StackDescription$ Move any number of counters from target permanent you control onto another target permanent.

```

Comment:
`MoveCounter  | Cost$` extra space after MoveCounter `MoveCounter | Cost$`

Reply:
Also the `Source$ ParentTarget` needs to be removed